pojo/src/main/java/com/spotify/hamcrest/pojo/MethodReference.java

```diff
@@ -21,10 +21,19 @@
 package com.spotify.hamcrest.pojo;
 
 import java.io.Serializable;
-import java.util.function.Function;
 
 /**
- * Adds serialization marker to {@code Function}.
+ * An interface for serializable method references. It is only valid to construct instances of this
```

Comment:
gotcha. I was going to suggest clarifying what will happen in this comment, but I think if it throws an IAE then that is clear enough.